README.md

```diff
@@ -3,6 +3,15 @@ Edit prototype in browser
 
 This is a plugin to the govuk-prototype-kit which adds an in-browser editor.
 
+Setup
+---
+
+To enable the editor you need to install the package and add the router.  You can do both of these with the following command:
+
+```shell
+npm install @x-govuk/edit-prototype-in-browser && echo "require('@x-govuk/edit-prototype-in-browser').addRoutes(require('govuk-prototype-kit').requests)" >> app/routes.js
```

Comment:
Does this combined command work on Windows?

Reply:
Ooh, good question.  I'll try it.

We've got two other options (that I can see):

1. `npx @x-govuk/edit-prototype-in-browser install`
2. Provide the install command and separately provide the line to put into `app/routes.js`

Reply:
I've updated the README to avoid using a chained command - that's something that can be looked into and refined later, it won't be needed if the Plugin Framework allows routers further down the line (and I've planned for that with my interface).